datastore/filesystem_datastore.nim

```diff
@@ -21,18 +21,17 @@ const
 
 proc new*(
   T: type FileSystemDatastore,
-  root = "data"): ?!T =
+  root = getCurrentDir()): ?!T =
```

Comment:
@dryajov `getCurrentDir()` seems like a reasonable default value, but would it be preferable to not have a default value?

Reply:
Based on discussion during today's team call, I have changed it so there's no default value for the `root` parameter.